packages/notebook/src/widget.ts

```diff
@@ -1194,6 +1195,17 @@ export class Notebook extends StaticNotebook {
     this.node.tabIndex = 0; // Allow the widget to take focus.
     // Allow the node to scroll while dragging items.
     this.node.setAttribute('data-lm-dragscroll', 'true');
+    /*this.addFooter();*/
+  }
+
+  /**
+   * Adds a footer to the notebook.
+   */
+  protected addFooter(): void {
+    const trans = this.translator.load('jupyterlab');
+    const info = new NotebookFooter(this);
+    info.node.textContent = trans.__('Click to add a cell.');
```

Comment:
You should set the content of the node inside the class `NotebookFooter`

Reply:
@trungleduc Thanks for the comment. This has been taken into account.